app/assets/javascripts/controllers/ansible_credentials/ansible_credentials_form_controller.js

```diff
@@ -0,0 +1,130 @@
+ManageIQ.angular.app.controller('ansibleCredentialsFormController', ['$http', '$window', 'credentialId', 'miqService', 'API', function($http, $window, credentialId,  miqService, API) {
+  var vm = this;
+
+  var init = function() {
+    vm.credentialModel = {
+      id: null,
+      name: '',
+      type: '',
+    };
+
+    vm.credential_options = {};
+    vm.select_options = [];
+
+    vm.newRecord = credentialId == 'new';
+    vm.afterGet = false;
+    vm.model = 'credentialModel';
+    ManageIQ.angular.scope = vm;
+    vm.saveable = miqService.saveable;
+
+    miqService.sparkleOn();
+
+    // get credential specific options for all supported credential types
+    API.options('/api/authentications')
+      .then(getCredentialOptions)
+      .catch(miqService.handleFailure);
```

Comment:
After https://github.com/ManageIQ/manageiq-ui-classic/pull/783 merge, the `.catch` will now catch the API failures, but those need to be visible to the user as flash messages.
Created a PR https://github.com/ManageIQ/manageiq-ui-classic/pull/805 that will display the error message flash.